frontend-project/src/typings.d.ts

```diff
@@ -28,9 +28,11 @@ interface Window {
   ) => void;
   reloadAuthorized: () => void;
 }
-
 declare let ga: Function;
 
+declare const BUILD_DATE: date;
+declare const BUILD_BRANCH: string;
+declare const BUILD_SHA: string;
```

Comment:
chyba też `string?`

Reply:
BUILD_DATE -> string ?

Reply:
możliwe, że to był powód, dla którego @ad-m tak zrobił, trzeba by go dopytać
jeżeli tamta wersja działała dobrze, to możesz zrobić git reset --soft do commita, który działał i tak jak pisałem przed edycją, w razie czego zrobimy z tego osobny PR, to wyjście wydaje mi się najlepsze

Reply:
tylko dodaj te pytajniki przy BUILD_BRANCH i BUILD_SHA, bo one zdaje mi się mogą być undefined, a BUILD_DATE (też chyba jest stringiem) nie

Reply:
`frontend-project/.eslintrc.js` wymaga adekwatnej zmiany, aby widział duże litery. Podobnie `frontend-project/config/config.ts`, aby ustawić wartości. Wszystko jest stringiem, co widać w `frontend-project/config/config.ts`.

Reply:
@ad-m już to znalazłem, w którymś komentarzu, problemem jest teraz tylko linter